README.md

```diff
@@ -4,6 +4,19 @@
 
 **Babelfish is a self-hosted server for source code parsing.** Babelfish can parse any file, in any language, extract an ASTs (Abstract Syntax Tree) from it, and convert it to a [**UAST**](./uast/specification.md) (Universal Abstract Syntax Tree).
 
+## Motivation and Scope
+
+Babelfish was born as a solution for massive code analysis. The goal: analyzing
+all source code from every repository, for every revision.
+
+Current **scope is parsing of single files in any programming language**
```

Comment:
Maybe better `Current scope is **parsing single files in any programming language**`?